charts/kong/README.md

```diff
@@ -201,6 +205,45 @@ The package to run can be changed via `image.repository` and `image.tag`
 parameters. If you would like to run the Enterprise package, please read
 the [Kong Enterprise Parameters](#kong-enterprise-parameters) section.
 
+### Separate admin and proxy nodes
+
+Users may wish to split their Kong deployment into multiple instances that only
+run some of Kong's services, e.g. where some nodes only run the proxy and other
+only run the admin API, or where some nodes only run Developer Portal services.
+At present, these require separate Helm releases (i.e. you run `helm install`
```

Comment:
Let's frame this as being a positive rather than a limitation.

Reply:
"At present" was meant more in the sense that it may change in the future, but isn't really necessary. I removed it, though that just leaves a neutral phrase. Not sure of any particular way to cast it in a positive light (mainly because we don't really have concrete alternatives to compare it to).